src/App.jsx

```diff
@@ -23,23 +23,42 @@ export const alex = {
   isMarried: false,
 };
 
-export const App = () => (
-  <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
+function Person({ name, age, sex, isMarried, partnerName }) {
+  let partner = '';
 
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
+  if (sex === 'm') {
+    partner = 'wife';
+  } else {
+    partner = 'husband';
+  }
```

Comment:
Use ternary operator
```suggestion
  const partnerStatus = sex === 'm' ? 'wife':  'husband'
```